cogs/blackjack/blackjack.py

```diff
@@ -0,0 +1,617 @@
+import discord
+import os
+import asyncio
+from discord.ext import commands
+from .utils.dataIO import fileIO
+from random import randint
+from .utils import checks
+from __main__ import send_cmd_help
+from cogs.economy import check_folders
+
+
+class Blackjack:
+    """Blackjack economy addition cog"""
+
+    def __init__(self, bot):
+        self.bot = bot
+        self.bank = fileIO("data/economy/bank.json", "load")
```

Comment:
don't edit file outside of it's own subfolder please.
you don't know what the future format of those files will be.
use the functions economy provides to edit it's data.